cpp/src/ray/runtime/task/local_mode_task_submitter.cc

```diff
@@ -48,12 +48,15 @@ ObjectID LocalModeTaskSubmitter::Submit(InvocationSpec &invocation,
   std::string task_name =
       invocation.name.empty() ? functionDescriptor->DefaultTaskName() : invocation.name;
 
+  rpc::JobConfig kDefaultJobConfig;
```

Comment:
nit:
make it a global const static variable?

not sure, but I always feel kXYZ is global const variables.